product_usability/models/product_attribute.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2022 Akretion (<http://www.akretion.com>).
+# @author Kévin Roche <kevin.roche@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+class ProductAttribute(models.Model):
+    _inherit = "product.attribute"
+
+    values_count = fields.Integer(compute="_compute_values_count")
+
+    @api.depends("value_ids")
+    def _compute_values_count(self):
+        for attr in self:
+            attr.values_count = len(attr.value_ids)
```

Comment:
Hi @Kev-Roche, I think it could be a good improvement if you execute the search query only once.
Above I suppose that `len(attr.value_ids)` execute a query for each iteration (so many in tree view).
You may problably search all value_ids at once outside for loop and store in a dict !?
Thanks